src/njoy21/input/MIXR/Card3/test/Card3.test.cpp

```diff
@@ -0,0 +1,126 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "MIXR, Card3",
+          "[MIXR] [Card3]" ){
+  GIVEN( "valid inputs" ){
+    WHEN( "valid values are provided" ){
+      std::string inp(
+ " 816 0.2 /\n"
```

Comment:
I don't think each line of Card3 ends with a `/`. I think only the card (can be multiple lines) is terminated with a `/`. Have you tested this in NJOY2016?

Reply:
Page 625 of the NJOY2016 manual has an example case that does have the forward slashes after every pair of (matn,wtn).

Reply:
I saw this after I posted that message.